tests/apps/test_clique.py

```diff
@@ -99,13 +99,6 @@ def test_expected_growth(self):
         assert result == [0, 1, 2, 3]
 
 
-def patch_random_shuffle(x, reverse):
-    """Dummy function for ``np.random.shuffle`` to make output deterministic. This dummy function
-    reverses ``x`` in place if ``reverse == True`` and does nothing otherwise."""
-    if reverse:
-        x.reverse()
-
-

```

Comment:
No longer required because how ties are settled in the `shrink` function has been updated to use `np.random.choice` over `np.random.shuffle`